### ISSUE TITLE
FoV currently missing

### ISSUE THREAD

**Issue by aboucaud** [aboucaud/galcheat#56]
I think the field of view would be a nice value to have in `galcheat`, even though not useful at the BTK level.

What do you think @ismael-mendoza, @HironaoMiyatake @mpaillassa ?

**Comment by aboucaud** [comment]
I mean the footprint of a single exposure of the survey on the sky.
We could also add the full area covered by the survey.

**Comment by ismael-mendoza** [comment]
In fact I have some notes from the BL meeting some months ago suggesting:
> Location for survey footprints, whether two surveys have any overlapping region? or some approximation to them (arcseconds)

**Comment by mpaillassa** [comment]
Yes I think it could be nice to have both. I propose a definition and different names for each:
- Area covered by a single frame/exposure or by the CCD mosaic: field of view, field size, field area.
- Area covered by the survey: survey area, survey size, survey (sky) footprint.

With the second we can then imagine having a small script to display the survey areas or compute overlap regions between surveys.
